### PR TITLE
i18n(zh-tw): update `ui.ts`

### PR DESCRIPTION
**Pull request by vrabe**
#### Description (required)

update `ui.ts`

#### Related issues & labels (optional)

- Relates to #7305 #9182 
- Suggested label: i18n

### CODE REVIEW

File: src/i18n/zh-tw/ui.ts [hunk 1]
```diff
@@ -126,6 +128,8 @@ export default UIDictionary({
 	'stub.description.cms': '知道更多在 Astro 使用此 CMS 的資訊？',
 	'stub.description.backend': '知道更多在 Astro 使用此後端服務的資訊？',
 	// Starlight banner
-	'starlight.title': '想要創建自己的文檔嗎？',
+	'starlight.title': '想要創建自己的文件嗎？',
```
Comment (mingjunlu) [inline src/i18n/zh-tw/ui.ts:131]:
```suggestion
	'starlight.title': '想要建立自己的文件嗎？',
```

nit: 「建立」可能更貼近臺灣的習慣，供參考～

Comment (vrabe) [reply src/i18n/zh-tw/ui.ts:131]: 對，用建立比較好

**Review by mingjunlu — APPROVED** [review]
LGTM!

Thanks for the updates, @vrabe. Welcome to Team Docs!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good job! 🙌

**Pull request merged**